### ISSUE TITLE
CrossSelector doesn't stretch

### ISSUE THREAD

**Issue by ahuang11** [pyviz/panel#499]
```
dirs_widget = pn.widgets.CrossSelector(
    name='Available Directories', value=[],
    options=['dir1', 'dir2'], sizing_mode='stretch_both')
dirs_input = pn.widgets.TextInput(
    name='dirs')

pn.WidgetBox(dirs_widget, dirs_input, sizing_mode='stretch_both')
```

![image](https://user-images.githubusercontent.com/15331990/60614509-dddb7580-9d81-11e9-99f4-1a66f6f32dc2.png)

Also the left/right margins for the <</>> buttons are uneven

**Comment by ahuang11** [comment]
@jlstevens

**Comment by jlstevens** [comment]
Thanks for filing this @ahuang11. Having a responsive width cross-selector this would also be helpful for one of my ongoing projects and I also noticed the problem with the uneven `<<` and `>>` button spacing.

**Comment by philippjfr** [comment]
That issue with spacing seems to be a regression. The ``sizing_mode`` issue is not too surprising to me since the widget was implemented before the layout refactor.

**Comment by jlstevens** [comment]
> The sizing_mode issue is not too surprising to me since the widget was implemented before the layout refactor.

Does that mean this should be relatively easy to fix? I am happy to use panel dev versions or even patch panel myself with a short diff if necessary...

**Comment by philippjfr** [comment]
Shouldn't be too hard, it just needs to forward the sizing_mode parameters appropriately to the sub-widgets, i.e. the multi-select widgets should expand but the buttons shouldn't.

**Comment by jlstevens** [comment]
I think this issue can now be closed given that #545 has been merged.
